packages/SqueakMaps-TiledMaps.package/SMATiledMapMorph.class/instance/getBoundingBoxFitting.and..st

```diff
@@ -0,0 +1,35 @@
+helping
+getBoundingBoxFitting: aFirstSMAMapLocationMarker and: aSecondSMAMapLocationMarker
+
+	| location1 location2 padding boundingBox point1 point2 boundingBoxResult |
+	location1 := aFirstSMAMapLocationMarker worldPos.
+	location2 := aSecondSMAMapLocationMarker worldPos.
+	padding := (location1 dist: location2) * 0.0. "TODO: Why does padding not work?"
```

Comment:
Habt ihr das irgendwo festgehalten, dass das noch TODO ist? In einem Issue oder so?

Reply:
[Hier](https://github.com/hpi-swa-teaching/SqueakMaps/issues/23#issuecomment-1131756368) hinzugefügt: